doc/man3/X509_STORE_CTX_new.pod

```diff
@@ -71,18 +71,29 @@ It must be called before each call to X509_verify_cert(), i.e. a I<ctx> is only
 good for one call to X509_verify_cert(); if you want to verify a second
 certificate with the same I<ctx> then you must call X509_STORE_CTX_cleanup()
 and then X509_STORE_CTX_init() again before the second call to
-X509_verify_cert(). The trusted certificate store is set to I<store>, the end
-entity certificate to be verified is set to I<x509> and a set of additional
-certificates (which will be untrusted but may be used to build the chain) in
-I<chain>. Any or all of the I<store>, I<x509> and I<chain> parameters can be
+X509_verify_cert().
+The trusted certificate store is set to I<trust_store> of type B<X509_STORE>.
+This may be NULL because there are no trusted certificates or because
+they are provided simply as a list using X509_STORE_CTX_set0_trusted_stack().
+The end entity certificate to be verified is set to I<target>,
+and a set of additional certificates may be provided in I<untrusted>,
+which will be untrusted but may be used to build the chain.
+Any or all of the I<trust_store>, I<target> and I<untrusted> parameters can be
```

Comment:
IIRC, if the target is NULL, then the certificate to verify is the first element on untrusted chain.  If my recollection is correct, this may be worth stating.

Reply:
Better check in the code :wink:
```
int X509_verify_cert(X509_STORE_CTX *ctx)
{
    SSL_DANE *dane = ctx->dane;
    int ret;

    if (ctx->cert == NULL) {
        ERR_raise(ERR_LIB_X509, X509_R_NO_CERT_SET_FOR_US_TO_VERIFY);
        ctx->error = X509_V_ERR_INVALID_CALL;
        return -1;
    }
```
So I've added:
```
Yet note that if I<target> is NULL the verification target will need to
be set using X509_STORE_CTX_set_cert() before L<X509_verify_cert(3)> is invoked.
```

Reply:
Yet the default target of `X509_verify_cert()` you mention would make much sense.
I've just added that.